layouts/DocsPage/Header.tsx

```diff
@@ -23,10 +23,25 @@ interface DocHeaderProps {
 const GITHUB_DOCS = process.env.NEXT_PUBLIC_GITHUB_DOCS;
 
 const handleClickScroll = () => {
-  const element = document.getElementById("feedback");
+  const element = document.getElementById("feedbackContainer");
```

Comment:
Is there an alternative to `getElementByID` here that's more idiomatic to React?